repositories/src/main/scala/cromwell/pipeline/datastorage/dto/ProjectSearch.scala

```diff
@@ -0,0 +1,68 @@
+package cromwell.pipeline.datastorage.dto
+
+import play.api.libs.json._
+
+sealed trait ProjectSearchFilter
+object ProjectSearchFilter {
+  implicit lazy val projectSearchFilterReads: Reads[ProjectSearchFilter] = Reads[ProjectSearchFilter] { json =>
+    (json \ "type").validate[String].flatMap {
+      case "all"            => json.validate[AllFilter.type]
+      case "name"           => json.validate[ByNameFilter]
+      case "files"          => json.validate[ByFilesFilter]
+      case "configurations" => json.validate[ByConfigFilter]
+      case "OR"             => json.validate[ComplexOrFilter]
+      case "AND"            => json.validate[ComplexAndFilter]
```

Comment:
it should be `or` and `and` (i.e. lowercased), isn't it?

Reply:
Fixed